### DISCUSSION TITLE
Upload file

### DISCUSSION THREAD

**Discussion by bfrax** [DirectoryLister/DirectoryLister#1121] [General]
Hello,
Is it possible to upload file with directoryLister?
Many thanks

**Comment by PHLAK** [comment]
This is not possible.

**Comment by kaerbr** [comment]
Will there be a upload / delete feature or is your goal to keep this application as read only?

**Reply by PHLAK** [reply]
No, there is no plan to implement file uploads or deletes. Directory Lister will only ever be "read only" as you say.

**Reply by kaerbr** [reply]
Thanks for the clarification! :)

**Reply by varadins** [reply]
It would be great if you reconsidered.
